server/planarserver.py

```diff
@@ -129,16 +122,140 @@ async def start_servers():
     print()
 
 
-main_app.on_shutdown.append(on_shutdown)
+def server_main(_args):
```

Comment:
This is seemingly not used

Reply:
This still needs to be addressed (I'm talking about the `_args` if that was not clear

Reply:
Sorry, Tried to respond to this, but been a while since I used GitHub's reviews. 

_args isn't used in this function, but it is going to be provided, by line 263, so it needs to be in the signature. Variable names starting with an `_` not being used is a convention used in many linters, so I figured that would make it clear. 
Any suggestions on how I could make it more clear that it is not being used, but needed? I could rename it to just `_` or `_unused`

Reply:
I'd have to add some manual parsing logic in the case that the script is called without a sub command, since right now it's just providing None, but that could be done. I'll try to make that update this evening, and see how that works.

Reply:
Nah don't worry about it, if it's going to add a lot of extra logic, I'll just merge it as is.